openadapt/finetune.py

```diff
@@ -0,0 +1,52 @@
+from openadapt import crud, models
+from loguru import logger
+import fire
+
+VALID_MODELS = ("davinci", "mpt-7b")
+
+
```

Comment:
```suggestion
@logger.catch(reraise=True, onerror=lambda _: sys.exit(1))
```
this might be nice :)

Reply:
I'm not sure what this does, could you clarify?

Reply:
loguru will catch any errors raised in the function and output them with more detail